package.json

```diff
@@ -75,5 +75,9 @@
   "engines": {
     "node": ">=14"
   },
-  "sideEffects": false
-}
+  "sideEffects": false,
+  "volta": {
+    "node": "14.18.2",
```

Comment:
Could this be upgraded to 16 version please

Reply:
@harshith-venkatesh I've made the change!
please check it out.